nettle/3.0-dlopen-test-msys-test.patch

```diff
@@ -0,0 +1,11 @@
+--- nettle-3.2/testsuite/dlopen-test.c.orig	2016-05-16 12:09:42.904653700 -0400
++++ nettle-3.2/testsuite/dlopen-test.c	2016-05-16 12:13:04.290831000 -0400
+@@ -9,7 +9,7 @@
+ main (int argc UNUSED, char **argv UNUSED)
+ {
+ #if HAVE_LIBDL
+-  void *handle = dlopen ("../libnettle.so", RTLD_NOW);
++  void *handle = dlopen ("../msys-nettle-6.dll", RTLD_NOW);
```

Comment:
Hi@JPeter, can you change this to:

```
#ifdef __MSYS__
  void *handle = dlopen ("../msys-nettle-6.dll", RTLD_NOW);
#else
  void *handle = dlopen ("../libnettle.so", RTLD_NOW);
#endif
```

.. as it makes the patch not break other systems (and therefore more upstreamable).